CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.20 FATAL_ERROR)
```

Comment:
```suggestion
cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
```
To be consistent with ROOT

Reply:
Maybe you confused that with Cling? For ROOT the minimum is 3.20:
https://github.com/root-project/root/blob/master/CMakeLists.txt#L7

Reply:
Oh! OK, my bad! OK to go then